src/VisualStudio/Core/Def/Implementation/InheritanceMargin/MarginGlyph/InheritanceMargin.xaml

```diff
@@ -215,7 +84,172 @@
+                                                    <ItemsPresenter KeyboardNavigation.TabNavigation="Cycle" KeyboardNavigation.DirectionalNavigation="Cycle"
+                                                                    SnapsToDevicePixels="{TemplateBinding SnapsToDevicePixels}" Grid.IsSharedSizeScope="true"/>
+                                                </ScrollViewer>
+
+                                                <Border x:Uid="Border_6"
+                                                        BorderBrush="{DynamicResource {x:Static vsui:EnvironmentColors.CommandBarMenuBorderBrushKey}}"
+                                                        BorderThickness="1" />
+                                            </Grid>
+                                        </vsui:SystemDropShadowChrome>
+                                    </Popup>
+                                </Grid>
+                                <ControlTemplate.Triggers>
+                                    <Trigger Property="Icon" Value="{x:Null}">
+                                        <Setter TargetName="Icon" Property="Visibility" Value="Collapsed"/>
+                                    </Trigger>
+                                    <Trigger Property="IsHighlighted" Value="true">
+                                        <Setter TargetName="Bg" Property="Fill" Value="{DynamicResource {x:Static vsui:EnvironmentColors.CommandBarMenuItemMouseOverBrushKey}}"/>
+                                        <Setter TargetName="Bg" Property="Stroke" Value="{DynamicResource {x:Static vsui:EnvironmentColors.CommandBarMenuItemMouseOverBorderBrushKey}}"/>
+                                        <Setter Property="TextElement.Foreground" Value="{DynamicResource {x:Static vsui:EnvironmentColors.CommandBarMenuItemMouseOverTextBrushKey}}"/>
+                                        <Setter TargetName="Arrow" Property="Fill" Value="{DynamicResource {x:Static vsui:EnvironmentColors.CommandBarMenuMouseOverSubmenuGlyphBrushKey}}"/>
+                                    </Trigger>
+                                    <Trigger Property="IsEnabled" Value="false">
+                                        <Setter Property="Foreground" Value="{DynamicResource VsBrush.CommandBarTextInactive}"/>
+                                        <Setter TargetName="Arrow" Property="Fill" Value="{DynamicResource {x:Static vsui:EnvironmentColors.CommandBarTextInactiveBrushKey}}" />
+                                    </Trigger>
+                                </ControlTemplate.Triggers>
+                            </ControlTemplate>
+                        </Setter.Value>
+                    </Setter>
+                </Style>
+
+                <!-- Style used to show the header  -->
+                <Style x:Key="HeaderMenuItemStyle" TargetType="{x:Type MenuItem}">
```

Comment:
This is a new style applies to the header of the menu items.
e.g.
![image](https://user-images.githubusercontent.com/24360909/115774861-f48ff980-a366-11eb-918b-b5d6f4ac7b3e.png)

The margin, width & height are copied from the other style aboded.

The background &  font color is something I am not 100% sure, because here we want to have a different color comparing the normal menu item, indicate it is not clickable.
Here is how it looks like in different theme.
Light:

![image](https://user-images.githubusercontent.com/24360909/115775276-8ac41f80-a367-11eb-9e27-21fb336cbcdd.png)

Dark:
![image](https://user-images.githubusercontent.com/24360909/115775454-c0690880-a367-11eb-9c3f-237ff7bbf723.png)

Blue:
![image](https://user-images.githubusercontent.com/24360909/115775516-d4ad0580-a367-11eb-8722-462551bdf272.png)

(The blue one looks a little bit odd?)
If there are better color then that would be great

Reply:
Yea the blue ones are odd, but may match the theme (I don't use blue theme...)

Reply:
i kinda like it tbh :)